src/Scan/Scans.ts

```diff
@@ -50,8 +53,10 @@ export enum TestType {
   SQLI = 'sqli',
   SSRF = 'ssrf',
   SSTI = 'ssti',
+  STORED_XSS = 'stored_xss',
   UNVALIDATED_REDIRECT = 'unvalidated_redirect',
   VERSION_CONTROL_SYSTEMS = 'version_control_systems',
+  WEBDAV = 'webdav',
```

Comment:
Found the additional test type which was missing at CLI, but supported by Bright service. It's a part of default test set, when customer run scan without specifying the tests set.